shadow.c

```diff
@@ -117,10 +118,19 @@ static PHP_GSHUTDOWN_FUNCTION(shadow)
 }
 /* }}} */
 
+/* {{{ shadow_deps
+ */
+static const zend_module_dep shadow_deps[] = {
+	ZEND_MOD_OPTIONAL("zip")
```

Comment:
As far as I understand, the additional extensions do not include shadow, since it's not installed at the moment of running `make test`. It's some other piece of code which puts (all) additional extension before shadow (which is being tested). This is what we need to fix IMO.

We could try syncing up on this offline.